index.adoc

```diff
@@ -1,178 +1,101 @@
 = WildFly Documentation
+:toc:
+:wildfly-latest-version: 35.0.0.Final
+:wildfly-latest-major: 35
 
-== Releases
+== WildFly Application Server
 
-|===
-|WildFly Release | Jakarta EE Version
+WildFly is a powerful, modular, & lightweight application server that helps you build amazing enterprise Java applications.
 
-|link:35[35.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
+Learn more about *link:{wildfly-latest-major}[WildFly {wildfly-latest-version}]* (or link:./archives[previous releases]) and read the link:https://www.wildfly.org/guides/[Guides].
 
+WildFly provides Jakarta EE APIs that are documented in:
 
-|===
+* https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
+* https://jakarta.ee/specifications/platform/11/apidocs/[Jakarta EE 11]
 
-|===
-|Bootable JAR
 
-|link:bootablejar[Bootable JAR]
+== Build Applications With WildFly
 
-|===
+=== WildFly Maven Plugin
 
-|===
-|Galleon
+The WildFly Maven plugin allows you to manipulate a WildFly server directly from a Java Maven project.
+Its features include:
 
-|link:galleon[Galleon]
+* Deploy, redeploy or undeploy your application
+* Add resources
+* Execute CLI commands
+* Run a standalone server within Maven
+* Provision & package a WildFly server fit for your application
+* Build a container image with WildFly and your application
+* Build a Bootable Jar application
 
-|===
+Learn more about the link:wildfly-maven-plugin[WildFly Maven Plugin].
 
-|===
-|WildFly Galleon Plug-ins
+=== WildFly Glow
 
-|link:galleon-plugins[WildFly Galleon Plug-ins]
+WildFly Glow is a tool and a Maven plugin to automatically discover an application's requirements and provision WildFly with the right set of capabilities.
 
-|===
+Learn more about link:wildfly-glow[WildFly Glow].
 
-|===
-|WildFly Maven Plugin
+=== Bootable JAR
 
-|link:wildfly-maven-plugin[wildfly-maven-plugin]
+[NOTE]
+The WildFly Maven Plugin now incorporates the ability to produce Bootable JAR applications and is the recommended way to do it.
 
-|===
+Bootable JAR is a Maven plugin to build bootable JAR applications.
 
-|===
-|WildFly Glow
-
-|link:wildfly-glow[wildfly-glow]
-
-|===
-
-|===
-|Prospero
-
-|link:prospero[Prospero]
-
-|===
+Learn more about link:bootablejar[Bootable Jar].
```

Comment:
This is not a comment about this PR. I wonder if we should see how we could integrate the bootable JAR documentation into the wildfly-maven-plugin. I'm not sure how that would look, but it's good documentation and I don't think we want to lose it once he `wildfly-jar-maven-plugin` goes away.

Reply:
+1, we should grow the wildfly-maven-plugin beyond what is generated by Maven and includes the Bootable Jar doc.